climada/hazard/storm_europe.py

```diff
@@ -60,10 +60,10 @@
 
 class StormEurope(Hazard):
     """A hazard set containing european winter storm events. Historic storm
-    events can be downloaded at http://wisc.climate.copernicus.eu/ and read
+    events can be downloaded at https://cds.climate.copernicus.eu/ and read
```

Comment:
I would replace this with this DOI https://doi.org/10.24381/cds.9b4ea013 althought I must admit that I haven't tested those NetCDFs...

Reply:
Also, I would explicitly mention the synthetic data at https://doi.org/10.24381/cds.ce973f02